lib/cli_common/cli_common/phabricator.py

```diff
@@ -20,6 +21,12 @@
 logger = log.get_logger(__name__)
 
 
+class BuildState(enum.Enum):
```

Comment:
Since you added this, can you make use of it in update_lint_results, update_coverage_results and at https://github.com/mozilla/release-services/blob/538df4943053bd0c76576af89aa4a6d3947fac2f/src/codecoverage/bot/code_coverage_bot/phabricator.py#L142 ?